web/src/components/FilingsCalendar.test.tsx

```diff
@@ -111,4 +135,119 @@ describe("<FilingsCalendar />", () => {
       )
     ).toBeInTheDocument();
   });
+
+  it("displays filings calendar as list in mobile", () => {
+    setTabletScreen(false);
+
+    const dueDate = getCurrentDate().add(2, "months");
+    const annualReport = generateTaxFiling({
+      identifier: "annual-report",
+      dueDate: dueDate.format("YYYY-MM-DD"),
+    });
+
+    const userData = generateUserData({
+      taxFilingData: generateTaxFilingData({ filings: [annualReport] }),
+      preferences: generatePreferences({ isCalendarFullView: true }),
+    });
```

Comment:
All of this setup (due date, annual report, user data, operate references) are repeated and reused exactly across all these tests. I think it could be pulled out into a beforeEach to clean up and be less repetitive